README.rst

```diff
@@ -14,8 +14,8 @@ please!
 
 This is all licensed under the terms of the `Creative Commons Zero`_ license.
 
-Read this in `简体中文`_ (simplified Chinese). NOTE: this has not been reviewed
-by the alex/what-happens-when maintainers.
+Read this in `简体中文`_ (simplified Chinese) or in `Русском`_ (Russian).
+NOTE: this has not been reviewed by the alex/what-happens-when maintainers.
```

Comment:
Could you change this to be "NOTE: These translations have not been reviewed by the alex/what-happens-when maintainers."?